Fixes so that setting pins work. Adding function to set load on a pin.

MattisLind opened the pull request:
A fix so that --pin and --pin-state workd. Would throw an assertion error since the hal was net setup properly.

Added a way of setting load on a pin.

anders-bzn commented:
Please check indentation, looks like the code is intended by three spaces  instead of four?

MattisLind commented:
I think I fixed all indent problems. Apparently the emacs settings wasn't very good and haven't figured out how to set them correctly. Now all tabs are removed and replaced with four spaces.
